src/main/java/com/mixpanel/android/mpmetrics/DecideMessages.java

```diff
@@ -175,6 +175,14 @@ public synchronized InAppNotification getNotification(int id, boolean replace) {
         return notif;
     }
 
+    // if a notification was failed to show, add it back to the unseen list so that we
+    // won't lose it
+    public synchronized void markNotificationAsUnseen(InAppNotification notif, boolean isDebugMode) {
```

Comment:
If isDebugMode is coming from MPConfig.DEBUG we do not need to pass that parameter always. We can just check it here.